docs/admin/admin-ui/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+- installation
+- license
+---
+
+# Overview
+
+Gluu Flex Admin UI is a reactive web interface to simplify the management and configuration of your Auth Server. It is one of the key services of Gluu Flex that enables viewing and editing configuration properties, interception scripts, clients, users, metrics, etc at one place.
+
+## Installation
+
+Gluu Flex can be installed using [VM installer](../../install/vm-install/vm-requirements.md) or using [Rancher](../recipes/getting-started-rancher.md) on Cloud Native.
+
+During installation, we need to provide a Software Statement Assertion (SSA) which is used by Admin UI to register an OIDC client to access license APIs. Check the following [guide](../../install/software-statements/ssa.md) for the steps to issue SSA from the Agama Lab web interface.
+
+![image](../../assets/admin-ui/install-ssa.png)
+
+## Flex License
+
+After installation Admin UI can be accessed on `https://hostname/admin` (the `hostname` is provided during setup). This web interface can be accessed only after submitting a valid `license key` issue from Gluu.
+
+User can also generate a 30 days trial license. To enjoy long time uninterrupted access to Admin UI get a flex license-key issued from Gluu.     
```

Comment:
@duttarnab Consider running all the content through the grammar and spellcheck as a standard practice. We all have our favorite grammatical mistakes.

Reply:
used Grammarly's free version for sentence correction and made changes.